README.md

```diff
@@ -32,10 +32,11 @@ Current categories include:
 
 ### Criteria for adding projects to the page
 
-At this time, only projects that Red Hat employees are required to work on as part of their job will be listed. There are so many amazing projects that Red Hatters work on as enthusiasts, too many to recognize in one place. This criteria may be subject to change as the page evolves.
+At this time, only projects that Red Hat employees are required to work on as part of their job will be listed. This includes projects Red Hat stewards as well as projects in which Red Hat is just one contributor among others. There are so many amazing projects that Red Hatters work on as enthusiasts, too many to recognize in one place. This criteria may be subject to change as the page evolves.
 
 ### What about projects not on GitHub?
-This list is just for Red Hat contributions to projects hosted on GitHub.  See https://community.redhat.com/software/ for an attempt at listing Red Hat's contributions to the broader universe of FLOSS projects.
+
+This list is just for Red Hat contributions to projects hosted on GitHub.  See https://community.redhat.com/software/ for more comprehensive listing of Red Hat's contributions to free and open source projects, regardless of where they are hosted.
```

Comment:
Grammar nit: "for more comprehensive listing" -> "for a more comprehensive listing"